.packit.yaml

```diff
@@ -0,0 +1,20 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+upstream_package_name: podman
+downstream_package_name: podman
+
+actions:
+  post-upstream-clone:
+    - "curl -O https://src.fedoraproject.org/rpms/podman/raw/main/f/podman.spec"
```

Comment:
So, this nagged at me in the first PR. Shouldn't this be **fNN** instead of `main`?
```
 ..../<BRANCH>  f36, f37, rawhide
```

Reply:
That may be the case right now, but I don't think it's guaranteed tomorrow  or next month. Is there a way to variableize that part of the URL? My head is spinning from reading packit docs.

Reply:
@edsantiago I'll check, but I'm gonna say we can afford to deal with that later as my rpm autobuilder job does take care of running git merge and cherry-picking of the latest commits if git merge fails, from rawhide to the release branches.

Reply:
I left them a note about spec file from additional branches. Let's see how that goes.